### PR TITLE
Feature/4615 task extend risk calculation for exposure history

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
store riskPerDay into Contact journal store every time a new risk gets processed
exiting unit tests will work with this update as well
made some preparations for new tests

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4615

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/MockDiaryStore.swift [hunk 1]
```diff
@@ -167,6 +182,7 @@ class MockDiaryStore: DiaryStoringProviding {
 	private var locations: [DiaryLocation] = []
 	private var contactPersonEncounters: [ContactPersonEncounter] = []
 	private var locationVisits: [LocationVisit] = []
+	private var risklevelPerDays: [RisklevelPerDay] = []
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/MockDiaryStore.swift:185]:
```suggestion
	private var risklevelPerDays: [RiskLevelPerDay] = []
```

File: src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/RisklevelPerDay.swift [hunk 2]
```diff
@@ -0,0 +1,12 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+@testable import ENA
+
+struct RisklevelPerDay {
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/RisklevelPerDay.swift:8]:
```suggestion
struct RiskLevelPerDay {
```

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift [hunk 3]
```diff
@@ -446,7 +450,8 @@ extension RiskProvider {
 				mostRecentDateWithHighRisk: risk.details.mostRecentDateWithRiskLevel,
 				numberOfDaysWithLowRisk: risk.details.numberOfDaysWithRiskLevel,
 				numberOfDaysWithHighRisk: risk.details.numberOfDaysWithRiskLevel,
-				calculationDate: Date()
+				calculationDate: Date(),
+				riskLevelPerDate: [Date(): .low]
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift:454]: Shouldn't this be 
```suggestion
				riskLevelPerDate: [Date(): .high]
```
?

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift (hunk 3 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift:454]: Good finding - I'll change it

**Pull request merged**